src/style.css

```diff
@@ -0,0 +1,133 @@
+* {
+  box-sizing: border-box;
+}
+
+.page {
+  font-family: "Roboto", sans-serif;
+}
+
+.card {
+  max-width: 200px;
+  background: #fff;
+  border: 1px solid #f3f3f3;
+  border-radius: 5px;
+  display: flex;
+  flex-direction: column;
+  padding: 32px 16px 16 px;
+}
+
+.card__img {
+  width: 160px;
+  height: 134px;
+  margin: 32px auto 0;
+}
+
+.card__name {
+  font-style: normal;
+  font-weight: 500;
+  font-size: 12px;
+  line-height: 18px;
+  color: #060b35;
+  margin: 40px 16px 4px;
+}
+
+.card__product-code {
+  font-style: normal;
+  font-weight: 400;
+  font-size: 10px;
+  line-height: 14px;
+  color: #616070;
+  margin-top: 0;
+  margin-bottom: 16px;
+  margin-left: 16px;
+}
+
+.card__stars {
+  display: flex;
+  justify-content: space-between;
+  margin-bottom: 24px;
+  margin-left: 16px;
+}
+
+.stars {
+  display: flex;
+}
+
+.stars__star {
+  background: url(./images/star.svg) center no-repeat;
+  height: 16px;
+  width: 16px;
+  border-radius: 0.5px;
+}
+
+.stars__star:not(:last-child) {
+  margin-right: 4px;
+}
+
+.stars--4 .stars__star:nth-child(-n+4) {
+  background-image: url(./images/star-active.svg);
+}
+
+.card__reviews {
+  font-style: normal;
+  font-weight: 400;
+  font-size: 10px;
+  line-height: 14px;
+  display: flex;
+  align-items: flex-end;
+  text-align: right;
+  color: #060b35;
+  margin: 0 16px 0 0;
+}
+
+.card__price {
+  display: flex;
+  justify-content: space-between;
+  margin-bottom: 16px;
+  margin-left: 16px;
+}
+
+.card__price-text {
+  font-style: normal;
+  font-weight: 400;
+  font-size: 12px;
+  line-height: 18px;
+  display: flex;
+  align-items: flex-end;
+  color: #616070;
+  margin: 0;
+}
+
+.card__sum {
+  font-style: normal;
+  font-weight: 700;
+  font-size: 16px;
+  line-height: 18px;
+  display: flex;
+  align-items: flex-end;
+  text-align: right;
+  color: #060b35;
+  margin: 0 16px 0 0;
+}
+
+.card__button {
+  display: flex;
+  font-weight: 700;
+  font-size: 14px;
+  width: 166px;
+  height: 40px;
+  align-items: center;
+  justify-content: center;
```

Comment:
Move justify-content under display: flex. It's better for readability to keep related rules together